control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -297,6 +297,17 @@ message RepublishVolumeRequest {
   bool reuse_existing_fallback = 6;
 }
 
+// Resize a volume to the requested size
+message ResizeVolumeRequest {
+  // uuid of the volume
+  google.protobuf.StringValue uuid = 1;
```

Comment:
We've been using optional string for new ones, as optional was added back to spec.

Reply:
ok, I'll use `string` type but it shouldn't be made `optional` as this field is mandatory, right?

Reply:
Oh right, indeed, because you were using StringValue I had assumed you wanted it optional, but yeah here doesn't make sense.